src/AutorestSwift/View Models/PropertyViewModel.swift

```diff
@@ -34,15 +34,17 @@ struct PropertyViewModel {
     let name: String
     let comment: ViewModelComment
     let type: String
+    // default value of the proeprty
     let defaultValue: ViewModelDefault
+    // default value of the property in the init(). Valid values are either nil (for optoinal property) or "" (i.e. not specfied for required property in init() method.)
     let initDefaultValue: String
```

Comment:
There's a typo in the second comment (optoinal), but I'm not sure what is the rationale here? If there's a default value, why not just put that in the init?